lib/avromatic/model/attributes.rb

```diff
@@ -154,8 +164,9 @@ def avro_field_options(field)
           options
         end
 
-        def add_serializer(field)
-          custom_type = Avromatic.type_registry.fetch(field)
+        def add_serializer(field, field_class)
+          # TODO: handle serializer for custom type within union
```

Comment:
Should never hit an error here, since the error for adding a coercer comes first, but done.